enhancements/node-tuning/pao-in-nto.md

```diff
@@ -0,0 +1,333 @@
+
+
+### CI
+
+PAO contains both unit and functional tests implemented using Ginkgo.
+
+Integration with NTO is currently TBD, please advise if anything special is needed.
+
+One of the test suites (4_latency) implements an opinionated latency testing suite to get comparable results from customers and to prevent configuration / command line invocation mistakes. Keeping it in PAO/NTO or moving it into cnf-tests is TBD.
+
+### Build pipeline
+
+PAO would inherit the NTO pipeline and all the images will be released via ART.
+
+### Releases
+
+PAO should be shipped as part of core OCP after NTO will contain the migrated code of PAO.
+PAO already obeys the OCP github PR rules and uses the same PR bot so nothing changes in the lifecycle.
+
+### API stability
+
+The `PerformanceProfile` CR uses an `performance.openshift.io/v2` API group which marks it as stable interface that needs to be maintained for about a year after official deprecation. We do not expect any change to the API as part of this proposal.
+
+### Bugs
+
+PAO bugs are currently tracked in The OpenShift Bugzilla component [Performance Addon Operator](https://bugzilla.redhat.com/buglist.cgi?component=Performance%20Addon%20Operator&list_id=12053428&product=OpenShift%20Container%20Platform)
+
+This should be kept as-is, the Telco team will own all the bugs. We might need to move the component under NTO or rename it.
+
+The bugzilla component is currently not part of OCP metrics and we can include it. TBD
+
+
+### User Stories
+
+Both the NTO and PAO functionality should be kept intact and no regressions are allowed. The user should not notice anything except the fact that OLM is no longer needed to enable the tuning functionality.
+
+### API Extensions
+The existing PAO API Extensions will be moved under the NTO repository.
+
+### Implementation Details/Notes/Constraints
+
+- Existing api compatibility practices:
+PAO API has been treated as Tier 1 for multiple releases already (since 4.4).
+
+- Enumerate list of performance profile owned objects:
+MachineConfig, KubeletConfiguration, Tuned (see [diagram](https://github.com/openshift-kni/performance-addon-operators/blob/master/docs/interactions/diagram.png)).
+All activity happening on the cluster or nodes is indirect via other components (MCO, NTO) reacting to changes in objects owned by PAO.
+
+- Enumerate list of webhooks:
+Validation and conversion webhook (TBD)
+
+- Updates to openshift conformance test suite if included by default:
+All basic checks are doable on all linux platforms except when real-time kernel is needed. That case used to be only supported on GCP (and bare metal).
+Measuring real latency requires bare metal, but is not needed for conformance.
+
+
+### Risks and Mitigations
+
+- Since the performance profile API will be visible for users on all clusters there is a risk that a cluster admin might tune nodes improperly.
```

Comment:
nit: As a mitigation we can say that we document clearly that admins proceed with caution.

Reply:
done